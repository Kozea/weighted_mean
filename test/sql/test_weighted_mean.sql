create extension weighted_mean;

create temp table test as (
  select a::numeric, b::numeric 
  from 
    generate_series(1, 100) as a(a),
    generate_series(1, 100) as b(b));

select weighted_mean(a,b) from test;

update test set b = 0;

select weighted_mean(a,b) from test;

