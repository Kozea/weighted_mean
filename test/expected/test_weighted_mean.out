create extension weighted_mean;
create temp table test as (
  select a::numeric, b::numeric 
  from 
    generate_series(1, 100) as a(a),
    generate_series(1, 100) as b(b));
select weighted_mean(a,b) from test;
    weighted_mean    
---------------------
 50.5000000000000000
(1 row)

